src/server/FantomasOnlinePreview/FantomasOnlinePreview.fsproj

```diff
@@ -1,26 +1,25 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
-    <OutputType>Exe</OutputType>
-    <AzureFunctionsVersion>v3</AzureFunctionsVersion>
-    <AssemblyName>FantomasOnlinePreview.Server</AssemblyName>
-    <RootNamespace>FantomasOnlinePreview.Server</RootNamespace>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <GenerateRuntimeConfigurationFiles>true</GenerateRuntimeConfigurationFiles>
+    <AWSProjectType>Lambda</AWSProjectType>
+    <OutputType>exe</OutputType>
+    <!-- This property makes the build directory similar to a publish directory and helps the AWS .NET Lambda Mock Test Tool find project dependencies. -->
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
   </PropertyGroup>
-  <ItemGroup>
-    <Content Include="host.json">
-      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
-    </Content>
-  </ItemGroup>
   <ItemGroup>
     <None Include="paket.references" />
     <Compile Include="../../shared/FantomasOnlineShared.fs" />
+    <Compile Include="..\HttpConstants.fs" />
```

Comment:
@nojaf I saw in multiple files that you are mixing backslashes with forward slashes ... I assume it just works fine but is seems a bit dirty, no?

Reply:
Yeah you have point here, but for this project I'll let it slide 🙈